[OCPBUGS-76467]IBM Z changes for ABI vlan and bond support (#106271)
openshift/openshift-docs: modules/configuring-network-overrides-ibm-z.adoc

[NOTE]
====
The `override` parameter overrides the host's network configuration settings.
====

[IMPORTANT]
====
The `ip=` kernel parameter uses the following syntax:

`ip=[IP]:[Gateway]:[Netmask]:[Hostname]:[Interface]:[None]:[DNS]`

For VLAN configurations:

* Define both the **base interface** and the **tagged VLAN interface** separately.
* The `vlan=` parameter links the tagged interface (for example, `encbdf0.300`) to the underlying physical interface (`encbdf0`).

For bonded interfaces:

* No changes are required in the default kernel command-line parameters.

> * Providing the appropriate bond configuration in the `agent-config` file is sufficient for nodes to be installed using bonded interfaces.

Reviewer: ```suggestion
* To install nodes by using bonded interfaces, provide the appropriate bond configuration in the `agent-config` file.
```
